SimG4Core/DD4hepGeometry/interface/DD4hep_DDDWorld.h

```diff
@@ -0,0 +1,30 @@
+#ifndef SIMG4CORE_DD4HEP_DDDWORLD_H
+#define SIMG4CORE_DD4HEP_DDDWORLD_H
+
+#include "G4VPhysicalVolume.hh"
+
+namespace cms {
+
+  class DDDetector;
+  
+  class DDDWorld {
+
+  public:
+    DDDWorld(const cms::DDDetector*);
+    ~DDDWorld();
+    static void SetAsWorld(G4VPhysicalVolume* pv);
+    static void WorkerSetAsWorld(G4VPhysicalVolume* pv);
```

Comment:
@ianna , it is the same world, why making different methods WorkerSetAsWorld(G4VPhysicalVolume* pv) and GetWorldVolumeForWorker() ? I would remove both.

Reply:
Good question, I’d gladly remove the code

Reply:
@civanch - all plugins ought to be in a plugin directory unrelated of their purpose. The Sim code is been copied as is to make sure that comparison is fair. I'll do a thorough cleanup at the later stage.